### PR TITLE
Switch from Glyphicons to FontAwesome

### PR DESCRIPTION
**Pull request by vxcamiloxv**
The 'pumpicon' class was added for long-term support in perzonalized styles or theme implementation in the future.

Closes: #1252

### CODE REVIEW

File: public/template/layout.jade [hunk 1]
```diff
@@ -10,13 +10,15 @@ html(lang="en")
...
       link(href="/stylesheet/bootstrap-responsive.min.css", rel="stylesheet")
+      link(href="/stylesheet/font-awesome.min.css", rel="stylesheet")
       link(href="/stylesheet/fineuploader.css", rel="stylesheet")
       link(href="/stylesheet/select2.css", rel="stylesheet")
 
     else
 
       link(href="//cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/2.3.2/css/bootstrap.min.css", rel="stylesheet", integrity="sha384-MY3lqAyZWEjvYefTFTBYUN85pnmwAxdSjsX+8IqCP1N5VBzOIX6GNo1rUt9gz74C", crossorigin="anonymous")
       link(href="//cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/2.3.2/css/bootstrap-responsive.min.css", rel="stylesheet", integrity="sha384-F6SkVapMeW8omBv9cyqV+15oQ6VSZ+g9B3dXePPOKKVGEdFOfLo50I4anpVkG3kQ", crossorigin="anonymous")
+      link(href="//cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css", rel="stylesheet", integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN", crossorigin="anonymous")
```
Comment (strugee) [inline public/template/layout.jade]: Where was Glyphicons originally being included from? We should remove that reference for perf

Comment (vxcamiloxv) [inline public/template/layout.jade]: Glyphicons is included in boostrap.css, could be removed and generate custom boostrap.css but the current cdn need change with this: https://maxcdn.bootstrapcdn.com/twitter-bootstrap/2.3.2/css/bootstrap-combined.no-icons.min.css

**Comment by vxcamiloxv** [comment]
@strugee Well, could be `picon`, but if you make this change LTS the class will not be needed

File: public/template/layout.jade (hunk 1 above)
Comment (strugee) [inline public/template/layout.jade]: Yeah, let's do that. Otherwise all the performance benefits we get from this are gone.

**Comment by strugee** [comment]
@vxcamiloxv oh, to be clear, I wasn't saying the class name seemed too long. I meant that having a separate class in general felt too heavyweight (i.e. I think we should just drop the secondary class altogether).

**Comment by vxcamiloxv** [comment]
Updated with latest changes

File: public/template/layout.jade [hunk 2]
```diff
@@ -8,15 +8,15 @@ html(lang="en")
 
     if config.noCDN
 
-      link(href="/stylesheet/bootstrap.min.css", rel="stylesheet")
-      link(href="/stylesheet/bootstrap-responsive.min.css", rel="stylesheet")
+      link(href="/stylesheet/bootstrap-combined.no-icons.min.css", rel="stylesheet")
```
Comment (strugee) [inline public/template/layout.jade]: Where is this `-combined` stylesheet coming from?

**Comment by strugee** [comment]
If you could also rebase, that would be great. Or I can do it for you if you don't have time

File: public/template/layout.jade (hunk 2 above)
Comment (vxcamiloxv) [inline public/template/layout.jade]: Is the same CDN but local: https://maxcdn.bootstrapcdn.com/twitter-bootstrap/2.3.2/css/bootstrap-combined.no-icons.min.css

Comment (strugee) [inline public/template/layout.jade]: Sorry, I meant, what is it? Is it a concatenation of the regular Bootstrap stylesheets and the `-responsive` stylesheets?

Comment (vxcamiloxv) [inline public/template/layout.jade]: Yes, but without Glyphicons

Comment (vxcamiloxv) [inline public/template/layout.jade]: Yes, but without Glyphicons

**Comment by vxcamiloxv** [comment]
Rebase with master

**Pull request merged**